translate_test.go

```diff
@@ -1993,6 +2327,94 @@ func TestTranslate3_3to3_2(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestTranslate3_4to3_3(t *testing.T) {
+	emptyConfig := types3_4.Config{
+		Ignition: types3_4.Ignition{
+			Version: "3.4.0",
+		},
+	}
+
+	_, err := v34tov33.Translate(emptyConfig)
+	if err != nil {
+		t.Fatalf("Failed translation: %v", err)
+	}
+
+	res, err := v34tov33.Translate(nonexhaustiveConfig3_4)
+	if err != nil {
+		t.Fatalf("Failed translation: %v", err)
+	}
+	assert.Equal(t, nonexhaustiveConfig3_3, res)
+
+	_, err = v34tov33.Translate(types3_4.Config{
+		Ignition: types3_4.Ignition{
+			Version: "3.4.0",
+		},
+		Storage: types3_4.Storage{
+			Luks: []types3_4.Luks{
+				{
+					Clevis: types3_4.Clevis{
+						Tang: []types3_4.Tang{
+							{
+								URL:           "http://example.com",
+								Thumbprint:    util.StrP("z"),
+								Advertisement: util.StrP("{payload: 'hello world'}"),
```

Comment:
It doesn't matter much, but this isn't valid JSON.

Reply:
Yeah enter lazy mode. I will fix that.